### ISSUE TITLE
Create a Guide/Template: Building a Brand Book through Team Exercises

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/knowledgebase-content#89]
### Overview
We need to create a guide for Brand Book so that teams can have consistent messaging for their project

### Action Items
The phases in the guide-making process are listed below. Each phase displayed in blue is linked to a wiki page with instructions on how to complete that phase. Open the wiki page in a new tab, copy the instructions for each part into the section labeled 'Tasks' at the bottom of this issue, and complete each task listed.
- [ ] [Gather Examples Solo](https://github.com/hackforla/guides/wiki/How-to-Gather-Examples:-Solo) (open in a new tab)
- [ ]  [Gather Examples Collaboratively](https://github.com/hackforla/guides/wiki/Phase-2-%E2%80%90-Gather-Examples-Collaboratively)
- [ ]  [Create a Guide Draft](https://github.com/hackforla/guides/wiki/How-to-Draft-a-Guide)
   - [ ]  If needed, create a template
- [ ]  [Have your guide peer reviewed](https://github.com/hackforla/guides/wiki/Peer-Reviews-for-Guides)
- [ ]  [Present your guide for leadership review](https://github.com/hackforla/guides/wiki/Submitting-Your-Guide-for-Leadership-Review)
### Resources/Instructions
- [GreenEarthOS](https://github.com/hackforla/climate-project)
  - [All issues related to their branding exercise](https://github.com/hackforla/GreenEarthOS/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature%3A+brand+exercise%22)
  - [Figma file](https://www.figma.com/file/JImygEUnM6m6giaP9qg3y4/GreenEarthOS-Branding-Exercise) where branding exercise is

### Resources
Update issue hackforla/UI-UX#15 with the name of item you are working on

#### Projects to check
- [100 Automations](https://github.com/100automations/website)
- [311 Data](https://github.com/hackforla/311-data)
- [Access the Data](https://github.com/hackforla/access-the-data)
- [BallotNav](https://github.com/hackforla/ballotnav)
- [Brigade Organizer's Playbook](https://github.com/codeforamerica/brigade-playbook)
- [Civic Tech Index](https://github.com/civictechindex/CTI-website-frontend)
- [Civic Tech Jobs](https://github.com/hackforla/CivicTechJobs)
- [Civic Tech Structure](https://github.com/hackforla/civic-tech-structure)
- [Data Science Projects](https://github.com/hackforla/data-science)
- [Design Systems](https://github.com/hackforla/design-systems)
- [Engage](https://github.com/hackla-engage)
- [Expunge-Assist (formerly known as Record Clearance)](https://github.com/hackforla/expunge-assist)
- [Food Oasis](https://github.com/hackforla/food-oasis)
- [GreenEarthOS](https://github.com/hackforla/climate-project)
  - [All issues related to their branding exercise](https://github.com/hackforla/GreenEarthOS/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature%3A+brand+exercise%22)
  - [Figma file](https://www.figma.com/file/JImygEUnM6m6giaP9qg3y4/GreenEarthOS-Branding-Exercise) where branding exercise is
<details>
<summary>Screenshot of brand exercise</summary>

![image](https://user-images.githubusercontent.com/37763229/171883282-c2164069-09eb-4e84-9330-1a9506209edb.png)
</details>

- [Guides](https://github.com/hackforla/guides) 
- [HomeUniteUs](https://github.com/hackforla/homeuniteus)
- [Internship project](https://github.com/hackforla/internship)
  - https://github.com/hackforla/internship/issues/68 (started with this issue)
  - https://github.com/hackforla/internship/issues/95 (epic for team contribution)
  - https://github.com/hackforla/internship/issues/81 (sample issue that should have been broken down into multiple issues)
  - https://github.com/hackforla/internship/issues/99
  - https://github.com/hackforla/internship/issues/98
  - https://github.com/hackforla/internship/issues/97
  - https://github.com/hackforla/internship/issues/96
  - https://github.com/hackforla/internship/issues/28
- [Lucky Parking](https://github.com/hackforla/lucky-parking)
- [Open Community Survey](https://github.com/hackforla/open-community-survey)
- [TDM Calculator](https://github.com/hackforla/tdm-calculator)
- [VRMS](https://github.com/hackforla/vrms)
- [Hack for LA's Website](https://github.com/hackforla/website)
- [Youth Justice Nav](https://github.com/hackforla/YouthJusticeNav/)

#### Taglines
Feeding America- A hunger-free America
Make-A-Wish: That people everywhere will share the power of a wish 
Habitat for Humanity: A world where everyone has a decent place to live
Airbnb- “Belong Anywhere.” 
HfLA Internship Project: Where Learning STEMS from Doing

#### This guide links
- [Brand Book folder](https://drive.google.com/drive/u/0/folders/1ydrihxFQmSzT0IKOgo9kBnVdudUvBzYI)
- [ ] This is where you will copy instructions from the wiki page for the step you are currently on.

**Comment by JesseTheCleric** [comment]
Would it be useful to save these related issues, or should I focus on finding specifically Brand Book excercises?
https://github.com/hackforla/311-data/issues/1951

**Comment by ExperimentsInHonesty** [comment]
Potential references
- URL: https://www.nngroup.com/articles/brand-intention-interpretation/
- why link is relevant: This article from a leading UI/UX design authority touches on the part of what Hack for LA includes in their brand book, by addressing the Brand Values (which Neilson Norman Group calls the "Brand promise") and Brand Attributes

**Comment by JesseTheCleric** [comment]
Documents Requesting Access for
 - [Expunge Assist One Sheet 2024](https://docs.google.com/document/d/1Xwbyumj7zm5jluCh2MWHUs2mXwU4xXNoUxgUuWTzGaE/)

**Comment by JesseTheCleric** [comment]
8-07 Progress Notes -
Starting at Civic Tech Jobs , moving down the list

**Comment by ExperimentsInHonesty** [comment]
Special thanks to Kelly Choe for the original Branding exercise and industry examples https://www.linkedin.com/in/kelly-choe/

**Comment by JesseTheCleric** [comment]
Documents requesting access too - 
-  https://docs.google.com/presentation/d/1XWY-OMiomzpKw_aCL1JkdgAhqpgGDZ2bu_XuReTO1zw/edit?slide=id.g183121c13a4_3_74#slide=id.g183121c13a4_3_74 this is to make comments for later, since we're refining the guide / issue handling process as well

**Comment by JesseTheCleric** [comment]
8-22-2025
Progress: Completed Phase 1 
Blockers: None so far
Availability: 4 Hours
ETA: 
Pictures or links :

**Comment by ExperimentsInHonesty** [comment]
Once you have looked through all the UI/UX issues that we identified, and found/documented all the relevant links from those issues, Bonnie can do this [UI/UX CoP drive search for "Brand"](https://drive.google.com/drive/search?q=brand%20in:0ACjCHzHKLLQxUk9PVA) to see if there is anything that was not caught

**Comment by jberzowska** [comment]
Thank you, that's great to hear. I'll just comment here to get updates.

**Comment by JesseTheCleric** [comment]
9-04-2025
Progress: Looked through UI/UX issues / files that mentioned brand identity. Beyond the group interview document that I saved in the draft folder, the vast majority of UI/UX dealt with logo iconography, colors and typography. Informal note - I asked my co-worker who was the UI/ UX lead at our company and she mentioned brand books were more of a marketing dept thing.
Blockers: None so far
Availability: 2-4 Hours 
ETA: Uncertain

**Comment by ExperimentsInHonesty** [comment]
The guide should convey 

Thing to know
The brand book exercises will give the needed context (Mission, Personality, Attributes, Brand Values) that will lead to developing a logo, choosing colors and main font style.  When all those elements are finished, the brand book will be complete.

**Comment by JesseTheCleric** [comment]
Questions for Bonnie -
It looks like many people did the same branding exercise, including you, is there any particular reason to break it up by person as opposed to one large issue or document? Most of the issues were empty or just linked back to the figjam

**Comment by JesseTheCleric** [comment]
9-18-2025
Progress : Finished looking through related Issues, documenting why each one is relevant
Blockers: None so far
Availability: 2-4 Hours 
ETA: Uncertain

**Comment by ExperimentsInHonesty** [comment]
Provided requested access to documents

> Request Access to - 
>  - May have incomplete branding exercise for Brigade Organizers Playbook  https://www.figma.com/file/clYtf0y30zYPhqLZ3AeB0a/BOP-Branding-Exercise?type=whiteboard&node-id=1%3A102&t=YKVFRLC9bckQpE0v-1

I copied this figjam to a new account for the guide. 
- [Knowledgebase figma account](https://www.figma.com/files/team/1550889024253402606/) - you can create a new figjam here
- [Copy of BOP Figjam](https://www.figma.com/board/UJ15IVUDMq9Jn0HcQTZY9P/copy-of-BOP?)


>  -  From Issue 68 ["Kelly's recording of what she expects from the brand personality exercise] (https://drive.google.com/file/d/1c4UE7_6EIOL9J_OLUCAnb6IO9Jo1iztx/view?usp=sharing)

I shared the internship drive with you so you can see the video.  Please watch it and decide if it has any value and let me know after so I can turn off access if not.

**Comment by JesseTheCleric** [comment]
9-26-2025
Progress : Spent time familiarizing self with FigJam, started on new branding template
Blockers: None so far
Availability: 2 Hours
ETA: Uncertain

**Comment by ExperimentsInHonesty** [comment]
ETA for a finished figjam and issue templates 20 hours

**Comment by JesseTheCleric** [comment]
4-2-2026
 - Reworded and  added additional voting areas under Brand personalities. 
 - What do we do in case of a tie?
- Mark where Brand attributes came from, which projects specifically
